Osnova.Net/Constants/DtfSubsiteIds.cs

```diff
@@ -0,0 +1,482 @@
+﻿namespace Osnova.Net.Constants
+{
+    static class DtfSubsiteIds
```

Comment:
Would be great to add source where to get subsites list as comments to these classes